src/Frontend/ContentContainer.php

```diff
@@ -71,9 +71,15 @@ public function apply_content_filter( $content ) {
 	 */
 	public function process_content( $post, $content ) {
 		$is_locked = get_post_meta( $post->ID, '_sesamy_locked', true );
-		$lock_mode = get_sesamy_setting( 'lock_mode' );
-		$preview   = apply_filters( 'sesamy_paywall_preview', static::extract_preview( $post ) );
-		$paywall   = apply_filters( 'sesamy_paywall', static::render_paywall() );
+		// If not locked, always use 'embed' lock mode
+		// Check if the value indicates locked content (truthy and not '0')
+		if ( empty( $is_locked ) || '0' === $is_locked ) {
```

Comment:
This is the actual change. It sets lock_mode to 'embed' if the article isn't locked